src/main/scala/org/apache/spark/sql/SqlUtils.scala

```diff
@@ -93,6 +94,15 @@ object SqlUtils {
           .map((x: AttributeReference) =>
             x.withQualifier(fullyQualifiedTableName.split('.').toSeq))
         )
+      case h: HiveTableRelation =>
+        h.copy(dataCols = h.dataCols
```

Comment:
why datacols and not output ?

Reply:
Unlike `LogicalRelation`'s output, `output` in HiveTableRelation is a method.

Reply:
oh ok.